apps/webpack.js

```diff
@@ -142,7 +142,8 @@ var karmaConfig = _.extend({}, baseConfig, {
       '@cdo/weblab/locale': path.resolve(__dirname, 'test', 'util', 'weblab', 'locale-do-not-import.js'),
       'firebase': path.resolve(__dirname, 'test', 'util', 'MockFirebase.js'),
       // Use mock-firmata to unit test playground-io maker components
-      'firmata': 'mock-firmata/mock-firmata'
+      'firmata': 'mock-firmata/mock-firmata',
+      'chrome-serialport': path.resolve(__dirname, 'test', 'unit', 'lib', 'kits', 'maker', 'FakeChromeSerialPort.js'),
```

Comment:
Rather than continually adding more stuff to this webpack config, I wonder if we should consider using this [babel-plugin-rewire](https://github.com/speedskater/babel-plugin-rewire).  What do you think?